doc/CMakeLists.txt

```diff
@@ -57,6 +57,9 @@ if (DOXYGEN_FOUND)
 
 	find_package(Perl)
 
+	add_executable (markdownlinkconverter markdownlinkconverter/markdownlinkconverter.c)
+	find_util (markdownlinkconverter MARKDOWN_LINK_CONVERTER "")
+
```

Comment:
> No, there is no possibility to pass arguments to a input filter in doxygen,

Ok, then it won't work with mingw. So there is no real reason to use `find_util`. Sorry for the unnecessary extra work.

> Why should cmake execute the markdownlinkconverter?

Its the description of the typical usage of `find_util` and does not apply to this situation.